http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -528,9 +532,105 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift](
       )
     } yield ()
 
-  def handleCreateAppMessage(msg: CreateAppMessage): F[Unit] = F.unit
+  def handleCreateAppMessage(msg: CreateAppMessage)(
+    implicit ev: ApplicativeAsk[F, AppContext]
+  ): F[Unit] = {
+    val createClusterAndNodepool =
+      for {
+        _ <- msg.cluster.fold(F.unit)(createClusterMessage => gkeInterp.createCluster(createClusterMessage, msg.appId))
```

Comment:
we should kick off some google operation sync to the message instead of queueing them